sdcm/cluster.py

```diff
@@ -340,6 +341,14 @@ def __init__(self, name, ssh_login_info=None, base_logdir=None, node_prefix=None
         self.replacement_node_ip = None  # if node is a replacement for a dead node, store dead node private ip here
         self._distro = None
 
+    @property
+    def running_nemesis(self):
+        return self._running_nemesis
+
+    @running_nemesis.setter
+    def running_nemesis(self, nemesis):
```

Comment:
I think the setter/getter can be removed now, if not needed and `running_nemesis` be assigned and used directly

Reply:
disagree, see comments below

Reply:
implement Lock in BaseNode